addons/mail/static/src/components/rtc_layout_menu/rtc_layout_menu.xml

```diff
@@ -0,0 +1,118 @@
+                        <div class="o_RtcLayoutMenu_radio_inputContainer">
+                            <input class="o_RtcLayoutMenu_radio_input" type="radio" name="filter" value="all" t-on-click="layoutMenu.onClickFilter" t-att-checked="!layoutMenu.callViewer.filterVideoGrid"/>
+                            <span class="o_RtcLayoutMenu_input_text">Show All</span>
+                        </div>
+                    </label>
+                    <label class="o_RtcLayoutMenu_radio">
+                        <div class="o_RtcLayoutMenu_radio_inputContainer">
+                            <input class="o_RtcLayoutMenu_radio_input" type="radio" name="filter" value="video" t-on-click="layoutMenu.onClickFilter" t-att-checked="layoutMenu.callViewer.filterVideoGrid"/>
+                            <span class="o_RtcLayoutMenu_input_text">Show only video</span>
+                        </div>
+                    </label>
+                    <hr class="o_RtcLayoutMenu_separator"/>
+                </t>
+                <label class="o_RtcLayoutMenu_radio">
+                    <div class="o_RtcLayoutMenu_radio_inputContainer">
+                        <input class="o_RtcLayoutMenu_radio_input" type="radio" name="layout" value="tiled" t-on-click="layoutMenu.onClickLayout" t-att-checked="messaging.userSetting.rtcLayout === 'tiled'"/>
+                        <span class="o_RtcLayoutMenu_input_text">Tiled</span>
+                    </div>
+                    <span class="o_RtcLayoutMenu_radio_icon">
+                        <t t-call="mail.RtcLayoutMenu_grid_svg"/>
+                    </span>
+                </label>
+                <label class="o_RtcLayoutMenu_radio">
+                    <div class="o_RtcLayoutMenu_radio_inputContainer">
+                        <input class="o_RtcLayoutMenu_radio_input" type="radio" name="layout" value="spotlight" t-on-click="layoutMenu.onClickLayout" t-att-checked="messaging.userSetting.rtcLayout === 'spotlight'"/>
+                        <span class="o_RtcLayoutMenu_input_text">Spotlight</span>
+                    </div>
+                    <span class="o_RtcLayoutMenu_radio_icon">
+                        <t t-call="mail.RtcLayoutMenu_spotlight_svg"/>
+                    </span>
+                </label>
+                <label class="o_RtcLayoutMenu_radio">
+                    <div class="o_RtcLayoutMenu_radio_inputContainer">
+                        <input class="o_RtcLayoutMenu_radio_input" type="radio" name="layout" value="sidebar" t-on-click="layoutMenu.onClickLayout" t-att-checked="messaging.userSetting.rtcLayout === 'sidebar'"/>
+                        <span class="o_RtcLayoutMenu_input_text">Sidebar</span>
+                    </div>
+                    <span class="o_RtcLayoutMenu_radio_icon">
+                        <t t-call="mail.RtcLayoutMenu_sidebar_svg"/>
+                    </span>
+                </label>
+            </t>
+        </div>
+    </t>
+
+    <!-- TODO TEMPORARY ICONS -->
```

Comment:
I made those icons because there was nothing alike in FA, they're good enough according to CPE maybe one day designers will replace them.

If we keep them, @phenix-factory will tell me "I told you".

Reply:
I told you !